docs/containers/index.md

```diff
@@ -1,10 +1,15 @@
-## Container Logging
+title: Sematext Container Observability
+description: Monitor Docker, rkt and Rancher container with Sematext by installing simple Agents and using our interface that shows everything in one simple screen.
 
-The [Logagent Docker image](https://hub.docker.com/r/sematext/logagent/) is tailored for the log collection on container platforms. It runs as a tiny container on every Docker host and collects logs for all cluster nodes and their containers. It is open-source agent created by Sematext.
+Some generic explanations...
```

Comment:
Did you mean to write something here?

Reply:
This file is not linked and currently hidden. I want to keep it in case we need it in the next Docathon iteration. I'll delete it afterward if it's not needed.